src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -439,7 +439,12 @@ private function _process_directives( string $html ) {
 					array_pop( $tag_stack );
 				}
 			} else {
-				if ( 0 !== count( $p->get_attribute_names_with_prefix( 'data-wp-each-child' ) ) ) {
+				$each_child_attrs = $p->get_attribute_names_with_prefix( 'data-wp-each-child' );
+				if ( ! is_countable( $each_child_attrs ) ) {
+					continue;
+				}
+
+				if ( 0 !== count( $each_child_attrs ) ) {
```

Comment:
I'm not 100% sure right now, but I think the `else` block below should execute when `$each_child_attrs` is `null`, shouldn't it?

At least that's the behavior in PHP < 8.0, when `count()` returned `0` if `null` is passed.


```suggestion
				if ( null !== $each_child_attrs && 0 !== count( $each_child_attrs ) ) {
```

Reply:
I think you're right. I thought it only contained a `foreach` with another `get_attribute_names_with_prefix()` but it contains another conditional block.

That other `get_attribute_names_with_prefix` has the same problem where it lacks a null check.